src/Blazor.Auth0.ClientSide/AuthenticationService.cs

```diff
@@ -25,13 +26,12 @@ namespace Blazor.Auth0
     public class AuthenticationService : IAuthenticationService, IDisposable
     {
         private readonly ClientOptions clientOptions;
-        private readonly IUriHelper uriHelper;
+        private readonly NavigationManager uriHelper;
```

Comment:
Whenever possible, please make the property/field/parameter match its base-type name, would you rename this parameter, please?